containers/LocationMarker.js

```diff
@@ -5,19 +5,48 @@ import { MapView } from 'expo';
 import PinCallout from '../components/PinCallout';
 
 class LocationMarker extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.handleDragStart = this.handleDragStart.bind(this);
+    this.handleDragEnd = this.handleDragEnd.bind(this);
+  }
+
+  handleDragStart(event) {
+    const { onDragStart, location } = this.props;
+
+    onDragStart(event, location.uploaded);
```

Comment:
Not sending the whole location to drag start?

Reply:
only really need to know whether it is uploaded or not so I didn't really see the need to send the whole object. Doesn't really matter either way as far as I can see.

Reply:
Cool.